op-bindings/Makefile

```diff
@@ -130,6 +131,9 @@ l1-blocknumber-bindings: compile
 dispute-game-factory-bindings: compile
 	./gen_bindings.sh contracts/dispute/DisputeGameFactory.sol:DisputeGameFactory $(pkg)
 
+fault-dispute-game-bindings: compile
+	./gen_bindings.sh contracts/dispute/FaultDisputeGame.sol:FaultDisputeGame $(pkg)
```

Comment:
We need a rewrite of `gen_bindings.sh` so bad, out of scope of this PR, i'll look into it eventually